### ISSUE TITLE
Error when running "make test-libc" in image simelotech/skycoin-dev:feature-stdevMauricio-1073

### ISSUE THREAD

**Issue by stdevIvr** [skycoin/skycoin#1481]
When executing "make test-libc" prompts up error messages like:

```sh
$ cc -o bin/test_libskycoin_shared lib/cgo/tests/*.c -lskycoin                    -lcriterion -lpthread -Ilib/cgo -Iinclude -Ibuild/usr/include -L build/libskycoin -Lbuild/usr/lib 
In file included from lib/cgo/tests/check_cipher.address.c:6:0:
/usr/local/include/criterion/new/assert.h:436:20: fatal error: stream.h: No such file or directory

In file included from lib/cgo/tests/check_cipher.crypto.c:6:0:
.......
In file included from lib/cgo/tests/check_cipher.hash.c:5:0:
.......
```

and some others.

**Comment by stdevMauricio** [comment]
Please try the following
`docker run -ti —rm -v ~/src/:/go/src/ simelotech/skycoin-dev:dev sh -c "cd skycoin; make test-libc"`

**Comment by stdevMauricio** [comment]
The error was introduced on May 2nd in the original repository. I made a fork, fixed the issue and now the Dockerfile uses it. Should be fixed in PR #1183
